docker/script/bootstrap.sh

```diff
@@ -42,6 +51,9 @@ then
   pip3 freeze > /requirements.txt
 else
   # flask-swagger depends on PyYAML that are known to be vulnerable
-  # even though Airflow names flask-swagger as a dependency, it doesn't seem to use it.
-  pip3 uninstall -y flask-swagger
+  # even though Airflow 1.10 names flask-swagger as a dependency, it doesn't seem to use it.
+  if [ "$AIRFLOW_VERSION" = "1.10.12" ]
```

Comment:
It will not be needed and is only there as a catch-all in case any user manually edits the docker compose to run a prev versions. If this were only a container image build we could have restricted this code to just the 1.10.12 branch and removed it from here.